contrib/systemd/crio-shutdown.service

```diff
@@ -6,8 +6,8 @@ Documentation=man:crio(8)
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/true
-ExecStop=mkdir -p /var/lib/crio; touch /var/lib/crio/crio.shutdown
+ExecStart=/usr/bin/rm -f /var/lib/crio/crio.shutdown
```

Comment:
Maybe move this under `/run` (which [must be cleared on reboot][1])?  Then you can stick with `true` in `ExecStart`.

[1]: http://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html

Reply:
I don't remember why /run did not work for us.  But I believe there was a reason.